.github/workflows/graph.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy Graph
+
+on:
+  push:
+    branches: master
+
+jobs:
+  deploy-goerli:
+    runs-on: ubuntu-latest
+    environment: graph
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: ./.github/workflows/setup-pnpm
+
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 18
+
+      - name: Codegen and Build
+        working-directory: ./packages/subgraph
+        run: |
+          pnpm codegen
+          cp subgraph.goerli.yaml subgraph.yaml
+          pnpm build
+
+      - uses: gtaschuk/graph-deploy@v0.1.12
+        with:
+          graph_access_token: ${{secrets.GRAPH_ACCESS_TOKEN}}
+          graph_subgraph_name: "balancer-pools-metadata"
+          graph_account: "bleu-llc"
+          graph_config_file: "./packages/subgraph/subgraph.yaml"
```

Comment:
Should we use `working-directory: ./packages/subgraph` here?

Reply:
Idk, does that work? 😬

Reply:
almost 😬... let's fix the block number